ttnn/cpp/ttnn/operations/reduction/generic/device/kernels/dataflow/reader_unary_transpose_wh_interleaved_input_cols_partitioned.cpp

```diff
@@ -36,23 +37,38 @@ void kernel_main() {
 
     uint32_t w = curr_col_in_batch;
 
-    // this reader will read a NHW tensor in NWH order
-    for (uint32_t i = 0; i < num_cols; i++) {
+    for (uint32_t i = 0; i < num_cols; i += row_chunk) {
```

Comment:
Please add a long comment about how this reads in data.
Need to include details about
- read a NHW tensor in N W_skip H W_chunk order
say what the difference is from NWH order
With an example (Ht=3, Wt=4, row_chunk=2 is the simplest I could think of), showing order of reading:
```
      Wt
Ht 0 1 6 7
     2 3 8 9
     4 5 10 11
```